srtcore/api.cpp

```diff
@@ -3475,6 +3538,9 @@ void* srt::CUDTUnited::garbageCollect(void* p)
 
 int srt::CUDT::startup()
 {
+#if HAVE_PTHREAD_ATFORK
+    pthread_atfork(NULL, NULL, (void (*)()) srt::CUDT::cleanupAtFork);
```

Comment:
If you register handlers for `pthread_atfork` in the startup, shouldn't you also unregister them (by calling `pthread_atfork(NULL, NULL, NULL)`) in the cleanup?